recipes/spyboat/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "spyboat" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 87f1c8889b46b01e47c0ccb7be12cbca00a1bd31bc90271a746ad58ca8e573bd
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - flit-core >=2.3
+  run:
+    - python
+    - numpy >=1.18    
+    - matplotlib >=3.1
```

Comment:
Unless you require a fully-functional GUI, it would be better to depend on `matplotlib-base`.

Reply:
pyboat comes with a GUI, and is also a requirement.. so in the end it needs to be on the system in any case. Shall I still change it?

Reply:
I guess the most important point is: matplotlib = matplotib-base + qt. So does the pyboat GUI use Qt?

Reply:
Yes, the it's a PyQT GUI!